src/pages/resultsView/enrichments/GenericAssayEnrichmentsTable.tsx

```diff
@@ -63,6 +69,15 @@ export default class GenericAssayEnrichmentsTable extends React.Component<
         this.props.dataStore.setHighlighted(d);
     }
 
+    private get entityTitle() {
+        return (
+            GENERIC_ASSAY_CONFIG.genericAssayConfigByType[
```

Comment:
this generic assay config is what we maintain for our instances?  and if some other instance is using GA, we derive it?

Reply:
Right now, I only derive one thing from serverConfig. You can assume `GENERIC_ASSAY_CONFIG` as a complex constant value. It stores customization for different generic assay data types (e.g. you can see we have lots of customization for `Methylation` type)

Reply:
We can derive all GENERIC_ASSAY_CONFIG from serverConfig in the future. I just think we should not do it in this round.